cnf-certification-test/lifecycle/suite.go

```diff
@@ -316,6 +322,15 @@ func testDeploymentScaling(env *provider.TestEnvironment, timeout time.Duration)
 	defer env.SetNeedsRefresh()
 	failedDeployments := []string{}
 	for i := range env.Deployments {
+		if scaling.IsManaged(env.Deployments[i].Name, env.Config.ManagedDeployments) {
+			if !scaling.CheckOwnerReference(env.Deployments[i].GetOwnerReferences(), env.Config.CrdFilters) {
+				failedDeployments = append(failedDeployments, env.Deployments[i].ToString())
+				tnf.ClaimFilePrintf("%s is scaling failed due to OwnerReferences that are not scalable", env.Deployments[i].String(), env.Deployments[i].GetOwnerReferences)
+			} else {
+				tnf.ClaimFilePrintf("%s is scaling skipped due to scalable OwnerReferences, test will run on te cr scaling", env.Deployments[i].String())
```

Comment:
This should probably be a `logrus.Infof` line.  We only want to print to the Claim file if there's an actionable error to fix/address.